internal/exec/stages/files/files.go

```diff
@@ -121,7 +121,9 @@ func (s *stage) relabeling() bool {
 func (s *stage) relabel(paths ...string) {
 	if s.toRelabel != nil {
 		for _, path := range paths {
-			s.toRelabel = append(s.toRelabel, filepath.Join(s.DestDir, path))
+			if !s.ToRelabelContains(path) {
```

Comment:
Good catch,
It was firstly intended not to set paths to relabel more than once as I've found code that do it multiple times.
I've noticed that relabeling was a bit long and that it could be done more than one time in units.go ...
This could be removed or commented for later commits because it's not relative to user-units

Reply:
removed